api/controller.py

```diff
@@ -122,6 +122,7 @@
 from config import (
     Configuration,
     CannotLoadConfiguration,
+    IntegrationException,
```

Comment:
It looks like this need to be imported for the REMOTE_INTEGRATION_FAILED problem detail to work?

Reply:
This is a mistake on my part. I tried a version of the code where trying to run a search when there was no search index configured would raise IntegrationException. What I ended up with was simpler but I didn't get rid of the import statement.